crates/cdk/src/cdk_lightning/mod.rs

```diff
@@ -121,16 +121,18 @@ pub struct PaymentQuoteResponse {
 }
 
 /// Ln backend settings
-#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq, Serialize, Deserialize)]
```

Comment:
> @prusnak I think i accidental delete your comment about removing copy. My response was showing up twice and i tried to delete the duplicate and deleted your comment as well I guess. 

Mint and Melt settings have `PaymentMethod` has one variant that is a custom string so it can't be copy but I think we can remove the custom or make it a static str and should be able to add copy back. Could make that its own pr or add it here I don't mind.

Reply:
No, it was me who deleted the comment. I figured out why the copy was removed, so I deleted it. I had no idea, you'd react so quickly :-)

What you write wrt `PaymentMethod` makes sense.

Reply:
https://github.com/cashubtc/cdk/pull/347